_sass/uno.scss

```diff
@@ -1108,10 +1107,12 @@ pre {
 
 .footer {
 	display: block;
-	padding: 2em 0 0 0;
-	border-top: 2px solid #dddddd;
+	padding: 17px 0 6px 0;
```

Comment:
Can these be em values?

Reply:
I think the footer issues are resolved now.

Reply:
![image](https://user-images.githubusercontent.com/4805803/64385536-8fbb4a80-d02e-11e9-9b59-458ef2a309dd.png)

Not quite there yet. This happens when the view switches to responsive